security-keycloak-authorization-quickstart/src/main/java/org/acme/security/keycloak/authorization/AdminResource.java

```diff
@@ -1,3 +1,18 @@
+/**
+ * Copyright 2019 Red Hat, Inc, and individual contributors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
```

Comment:
We don't add license headers in Quarkus. They shouldn't be added.

Reply:
good, I can do a cleanup in the other samples too.

Reply:
there are 14 others files with licence 

![image](https://user-images.githubusercontent.com/191879/140517269-2109108a-aefd-42e3-8cff-386f70b91d4d.png)